src/TailCommand.php

```
@@ -74,12 +74,12 @@ private function tailLocalLogFile()
     protected function tailRemoteLogFile($connection)
     {
         $connectionParameters = config('tail.connections.'.$connection);
-
+        $PORT = @$connectionParameters['port'] ?: 22;
```

Why are you trying to suppress errors here? 

Could you rename `$PORT` to `$port`?